user_stories/001.md

```diff
@@ -0,0 +1,11 @@
+# 001 - As a User, I want to securely log in to the learning management system
```

Comment:
```
As a user with valid credentials
I want to securely log in to the learning management system
So that I can have access to a personalized dashboard with the information about my courses
```

Reply:
Resolved on [e6b888c](https://github.com/wallidhgs/poc_bdd/pull/2/commits/e6b888ce657f9f06838dd598afe9863d51bda365)